bfd/elflink.c

```diff
@@ -13564,6 +13564,13 @@ _bfd_elf_section_already_linked (bfd *abfd,
 
   flags = sec->flags;
 
+  /* Check if the section type is SHT_COMDAT and the SEC_LINK_ONCE flag is not
```

Comment:
The section(s) are included for any file that uses JLI calls (the entry + stub). Only one copy of these sections (with the same name) should be included and the other discarded. The only way I see for an ELF file that has this behavior is the .gnu.linkonce.* sections; however, using this would change the names used as it may not be compatible with Metaware. If you have another suggestion on how to do this, please let me know.